flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -334,18 +334,32 @@
                     .stringType()
                     .noDefaultValue()
                     .withDescription(
-                            "A general option to probe Hadoop configuration through prefix 'flink.hadoop.'. Flink will remove the prefix to get <key> (from <a href=\"https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/core-default.xml\">core-default.xml</a> "
-                                    + "and <a href=\"https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/hdfs-default.xml\">hdfs-default.xml</a>) then set the <key> and value to Hadoop configuration. "
-                                    + "For example, flink.hadoop.dfs.replication=5 in Flink configuration and convert to dfs.replication=5 in Hadoop configuration.");
+                            Description.builder()
+                                    .text(
+                                            "A general option to probe Hadoop configuration through prefix 'flink.hadoop.'. Flink will remove the prefix to get <key> (from %s and %s) then set the <key> and value to Hadoop configuration."
+                                                    + " For example, flink.hadoop.dfs.replication=5 in Flink configuration and convert to dfs.replication=5 in Hadoop configuration.",
+                                            link(
+                                                    "https://hadoop.apache.org/docs/r2.4.1/hadoop-project-dist/hadoop-common/core-default.xml",
```

Comment:
Why did you change the link to Hadoop version 2.4.1 from stable?

Reply:
Thanks for your review again @tillrohrmann. 
Because I found that configuration option `yarn.application-attempts` is based on v2.4.1, so I continue to use 2.4.1 for alignment. If unbundling with a specific version is a better way, then I will use the most recent stable version instead, including `yarn.application-attempts`. 
If it's ok, I will update again.

Reply:
Yes, please update to link to the stable configuration.